examples/simple-platform/README.md

```diff
@@ -0,0 +1,39 @@
+# A Simple Interbit Platform 
+
+Before we dive into our Interbit platform example, let's take a step back and 
+define the ecosystem of Interbit applications. We are concerned with two types 
+of apps: Interbit platform apps and apps built on the platform. 
+
+By platform apps, we mean apps intended to make Interbit app development 
+easier by solving common problems such as user identity management, payment 
+processing, and hosting. These apps make up the Interbit platform. They may 
+not provide much value to the user on their own - after all, how useful is a 
+payment processing app if there is nothing to pay for? However, they allow 
+developers to concentrate their efforts on solving app-specific problems 
+instead of reinventing the wheel. Apps of the second type of are those built 
+by developers that use the functionality provided by the platform.
+
+This example has one platform app, `packages/app-account`, which provides user 
+authentication and identity management functionality. We also have a 
+[template app](../template.adoc), `packages/interbit-template`. This app 
+doesn't do much but is a starting point for creating your own Interbit 
+applications. 
+
+The goal of this example is to demonstrate how to request and share user 
+profile information from outside the accounts app. We configure the platform 
+and template app so that a user grants read access to their private accounts 
+chain from the template app. In order to demonstrate this, we also create a 
+user account that is authenticated via GitHub. Specifically, we go over:
+
+- the platform structure: configuration of the apps' chain joins and peers
+- chain hosting for our apps and the GitHub OAuth loop with 
+`packages/platform-deploy` and `packages/web-auth-endpoint`
```

Comment:
What are these? This is the first mention, and it's not clear if they are apps or something else. In `structure.md`, we call them hypervisors. We should do that here too, along with a mention of what a hypervisor is.